app/assets/javascripts/discourse/tests/helpers/create-pretender.js

```diff
@@ -39,7 +40,15 @@ const helpers = { response, success, parsePostData };
 
 export let fixturesByUrl;
 
-export default new Pretender();
+const instance = new Pretender();
+
+const oldRegister = instance.register;
+instance.register = (...args) => {
```

Comment:
Not a huge fan of this hack, but setting `baseUri` in `lib/get-url` (which is needed to get the right CDN URL) makes our `ajax` lib prepend the subfolder path to all the requests it makes which makes pretender unable to recognize our fixtures. One alternative fix is to add an `isTesting()` check in our `ajax` lib [here](https://github.com/discourse/discourse/blob/b32c7df0e9608863d2ff78b7823d636350a69428/app/assets/javascripts/discourse/app/lib/ajax.js#L169) and if we're in test mode then we skip the `getURL` call, but I'm not a fan of that approach either.